.lintstagedrc

```diff
@@ -0,0 +1,6 @@
+{
+  "*.{js,ts,tsx}": [
+    "eslint --fix",
+    "prettier --write"
+  ]
+}
```

Comment:
This configuration is telling lint-staged to execute ESLint and Prettier on the `.js`, `.ts`, or `.tsx` files.

It also tells these tools that they can fix the issues they find, if they are able to do so (`--fix` and `--write`)